src/tools/interop/idt/scripts/push.sh

```diff
@@ -0,0 +1,21 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+source idt/scripts/vars.sh
+mv idt/venv TEMPvenv
+scp -r ./idt/* "$PIUSER@$PIHOST:/home/$PIUSER"/idt
```

Comment:
It would be cleaner to use rsync with some `--exclude=PATTERN` options.

Reply:
Also, it looks like the `/home/kali/idt` doesn't exist yet on the target. Apparently, `rsync` can support that for you, while `scp` won't unless you are copying a directory instead of a group of files.

Reply:
https://github.com/project-chip/connectedhomeip/pull/29174#discussion_r1330805987